src/screens/AboutScreen.tsx

```diff
@@ -53,8 +62,11 @@ export class AboutScreen extends React.Component<Props> {
             return (
               <Text
                 fontSizeLargestScale={PV.Fonts.largeSizes.md}
+                isSecondary={!contributor.link}
                 key={`maintainers_${index}`}
-                onPress={() => this.handleFollowLink(contributor.link)}
+                {...(contributor.link ? {
+                  onPress: () => this.handleFollowLink(contributor.link)
+                } : {})}
```

Comment:
Just a readbility suggestion: Why not just update `handleFollowLink` to handle undefined? You don't need to change any of the props then to this syntax.

Reply:
If I pass a function into onPress, then the <Text> element has a TouchableOpacity type of effect, but we don't want that for names without a valid link/url.

I was surprised that having an onPress to Text has an opacity effect by default, but that seems to be the case in my tests.